### DISCUSSION TITLE
regarding updation of files

### DISCUSSION THREAD

**Discussion by Deviprashanth-Pazhanichami** [community/community#137227] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

how to input the datasets in GitHub

**Comment by Yackens** [comment]
To upload datasets to GitHub:

1. Create a Repository:
Go to GitHub, click the + icon, and select New repository.

2. Upload via Web:
Open your repository, click Add file > Upload files, select your dataset files, and Commit changes.

3. Upload via Git (Command Line):
Initialize Git, add files, commit, link to GitHub, and push:
```
git init
git add .
git commit -m "Add dataset"
git remote add origin https://github.com/yourusername/repo.git
git push -u origin master

```
4. Organize:
Optionally, create folders like data/ to organize datasets.

5. Include a README:
Add a README.md to describe your datasets.

If you found this answer useful, please, mark my answer as the correct one to help me obtain GitHub achievement: Galaxy Brain 😄

**Reply by ebndev** [reply]
Hi @Yackens, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers.

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by VALEK9292** [comment]
As

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Prashanth1234545 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
